libs/utils/utils.js

```diff
@@ -592,7 +592,7 @@ export async function loadArea(area = document) {
       loadGeoRouting(config, createTag, getMetadata);
     }
     const type = getMetadata('richresults');
```

Comment:
super nit: better var name - `richResults`?

Reply:
thx, addressed in https://github.com/adobecom/milo/pull/389/commits/6cec504e9e1558b4f77f96ccf947ad8acfd94c77